addons/starrocks-ce/templates/componentdefinition-be.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: {{ include "be.componentDefName" . }}
+  labels:
+    {{- include "starrocks.labels" . | nindent 4 }}
+spec:
+  provider: kubeblocks
+  description: A StarRocks BE v{{.Chart.AppVersion}} component definition for Kubernetes
+  serviceKind: starrocks-be
+  serviceVersion: {{.Chart.AppVersion}}
+  updateStrategy: BestEffortParallel
+  services:
+  - name: be
+    serviceName: be
+    spec:
+      ipFamilyPolicy: PreferDualStack
+      ipFamilies:
+        - IPv4
+      ports:
+      - name: be-http
+        port: 8040
+        targetPort: http-port
+      - name: proxy-http
+        port: 8080
+        targetPort: proxy-port
+  configs:
+  - name: be-cm
+    templateRef: {{ include "be.componentDefName" . }}-cm
+    namespace: {{ .Release.Namespace }}
+    volumeName: be-cm
+  volumes:
+  - name: data
+    needSnapshot: true
+  vars:
+  - name: FE_DISCOVERY_SERVICE_NAME
+    valueFrom:
+      serviceVarRef:
+        compDef: {{ include "fe.componentDefName" . }}
+        name: fe
+        host: Required
+  - name: MYSQL_PWD
```

Comment:
PASSWD is better, PWD is the abbreviation for 'Print Working Directory'.

Reply:
MYSQL_PWD is a native environment variable supported by the MySQL.